src/test/java/com/conveyal/datatools/manager/jobs/DeployJobTest.java

```diff
@@ -17,22 +18,26 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
+import java.util.ArrayList;
 import java.util.Date;
 import java.util.List;
 
+import static com.conveyal.datatools.TestUtils.getBooleanEnvVar;
 import static com.conveyal.datatools.manager.controllers.api.ServerController.getIds;
 import static com.conveyal.datatools.manager.controllers.api.ServerController.terminateInstances;
+import static com.zenika.snapshotmatcher.SnapshotMatcher.matchesSnapshot;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.Assert.assertFalse;
+import static org.junit.Assume.assumeTrue;
 
 /**
  * This contains a few helpful tests for quickly spinning up a deployment using any bundle of GTFS + OSM or pre-built
  * graph that already exists on S3.
  *
- * Note: these tests require credentials on the IBI AWS account, which is why the class is tagged with Ignore (so that it is
- * not run with the rest of the test suite)
+ * Note: some tests require credentials on the IBI AWS account, which is why those tests contain an assumeTrue statement
```

Comment:
Can this comment be modified slightly to suggest how to run those tests (e.g., set `RUN_AWS_DEPLOY_JOB_TESTS=true` to run these tests).